### PR TITLE
Add Ownbit wallet

### PR DESCRIPTION
**Pull request by jtomtan**
Add Ownbit wallet

### CODE REVIEW

**Comment by taylorjdawson** [comment]
@jtomtan Thanks for this! I am going to take a look and test today.

File: src/modules/select/wallets/ownbit.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { mobileWalletInstallMessage } from '../content'
+import {
+  WalletModule,
+  Helpers,
+  InjectedWithBalanceOptions
```
Comment (taylorjdawson) [inline src/modules/select/wallets/ownbit.ts:5]: @jtomtan `InjectedWithBalanceOptions` is an unused import, please remove 👍

File: src/modules/select/wallets/ownbit.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+} from '../../../interfaces'
+
+import ownbitIcon from '../wallet-icons/icon-ownbit'
+
+function ownbit(options: CommonWalletOptions): WalletModule {
```
Comment (taylorjdawson) [inline src/modules/select/wallets/ownbit.ts:10]: `CommonWalletOptions` needs to be imported

**Review by taylorjdawson — CHANGES_REQUESTED** [review]
Please run the following command to format your code according to the formatting guidelines 
```
npx prettier --write src/modules/select/wallets/ownbit.ts
```

File: src/modules/select/wallets/ownbit.ts (hunk 1 above)
Comment (jtomtan) [reply src/modules/select/wallets/ownbit.ts]: nice catch! 👍 InjectedWithBalanceOptions removed.

**Comment by taylorjdawson** [comment]
> Please run the following command to format your code according to the formatting guidelines
> 
> ```
> npx prettier --write src/modules/select/wallets/ownbit.ts
> ```

@jtomtan Almost there! Could you run the above command? The formatting is still off

**Comment by jtomtan** [comment]
> npx prettier

Now it's done. Thanks for your patience.

**Comment by taylorjdawson** [comment]
@jtomtan Thank you!

**Review by taylorjdawson — APPROVED** [review]

**Pull request merged**